resources/clients/react/item-tracker/src/config.json

```diff
@@ -1,3 +1,3 @@
 {
-  "BASE_URL": "http://REPLACE_WITH_YOUR_REST_ENDPOINT"
+  "BASE_URL": "http://localhost:8000/api"
```

Comment:
Remember to change this back to placeholder!
(We need a better solution for this, eventually...I'm sure it will keep happening to everyone.)

Reply:
Dang it!